doc/tools/pkcs11-tool.1.xml

```diff
@@ -714,30 +714,50 @@
 			in DER format from smart card:
 				<programlisting>pkcs11-tool --read-object --id KEY_ID --type cert --output-file cert.der</programlisting>
 
+			Write certificate to token:
+				<programlisting> pkcs11-tool --login --write-object certificate.der --type cert</programlisting>
+
 			To convert the certificate in DER format to PEM format, use OpenSSL
 			tools:
 				<programlisting>openssl x509 -inform DER -in cert.der -outform PEM > cert.pem</programlisting>
 
+			Generate new key:
```

Comment:
I would be more explicit here like "generate new RSA Key pair" to be obvious both of the keys are generated and to avoid confusion with generating symmetric key. Same below.

Reply:
Ok, I'll change it..